cluster-provision/manifests/calico.yaml

```diff
@@ -32,6 +32,9 @@ data:
           "ipam": {
               "type": "calico-ipam"
           },
+          "container_settings": {
```

Comment:
These manifests are merely copied from Calico. If we do an inline change, it is likely to be lost after we decide to update calico. Could you use patch for it?

Reply:
Well that's a pain to patch since is a json inside yaml field :-/

Reply:
You can sed this feild like we bo in provision scripts https://github.com/kubevirt/kubevirtci/blob/d244ff0a75e99803951bbf2d6a51b97b5859d116/cluster-provision/k8s/scripts/provision.sh#L119

Even add this logic to cni-map.sh::configure_calico() method.
https://github.com/kubevirt/kubevirtci/blob/d244ff0a75e99803951bbf2d6a51b97b5859d116/cluster-provision/common-scripts/config-cni.sh#L5

Reply:
After talking with @phoracek I am doing that, next push will be like that.